code/game/objects/effects/decals/Cleanable/fuel.dm

```diff
@@ -6,66 +6,74 @@
 	anchored = 1
 	var/amount = 1
 
-	New(turf/newLoc,amt=1,nologs=0)
-		if(usr && usr.client && !nologs)
-			message_admins("Liquid fuel has spilled in [newLoc.loc.name] ([newLoc.x],[newLoc.y],[newLoc.z]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[newLoc.x];Y=[newLoc.y];Z=[newLoc.z]'>JMP</a>)")
-			log_game("Liquid fuel has spilled in [newLoc.loc.name] ([newLoc.x],[newLoc.y],[newLoc.z])")
-		src.amount = amt
+/obj/effect/decal/cleanable/liquid_fuel/New(turf/newLoc,amt=1,nologs=0)
```

Comment:
This is just me implementing coding standards all over this file. Proper indentation and full paths